agent/wptdriver/webpagetest.cc

```diff
@@ -107,7 +107,7 @@ bool WebPagetest::GetTest(WptTestDriver& test) {
 
   // build the url for the request
   CString buff;
-  CString url = _settings._server + _T("work/getwork.php?shards=1&reboot=1");
+  CString url = _settings._server + _T("work/getwork.php?shards=1&reboot=1&multistep=1");

```

Comment:
Think at laud here:

Will a multi step measurement with 1 step be different then a measurement without the multistep option? If it is not the case, can we do without the "&multistep=1"? This would eliminate a code path in the WPT server.